package.json

```diff
@@ -6,6 +6,12 @@
     "eslint": "^4.1.1",
     "react": "^15.6.1",
     "react-dom": "^15.6.1",
+    "react-flexbox-grid": "^1.1.3",
+    "react-fontawesome": "^1.6.1",
```

Comment:
This component does not include any of the Font Awesome CSS or fonts, we need to make sure to include those on your end somehow, either by adding them to your build process or linking to